config/ses.toml

```diff
@@ -7,6 +7,7 @@ target = "devel7"
 
   [ses.ses7]
     rook_ceph_chart = "registry.suse.de/suse/sle-15-sp2/update/products/ses7/charts/rook-ceph:latest"
+    ceph_image = "suse.de/devel/storage/7.0/containers/ses/7/rook/ceph:latest"
```

Comment:
shouldn't this be registry.suse.de?

Reply:
Well, I copied it from  `[ses.ses7.yaml_substitutions]` 
There is also no `registry.suse.de`
Are both incorrect?

Reply:
Changed it to the correct value.